src/onboard_dataflowspec.py

```diff
@@ -652,17 +652,25 @@ def __get_silver_dataflow_spec_dataframe(self, onboarding_df, env):
 
             silver_target_format = "delta"
 
-            bronze_target_details = {
-                "database": onboarding_row["bronze_database_{}".format(env)],
-                "table": onboarding_row["bronze_table"]
-            }
+            if f"bronze_database_{env}" in onboarding_row:
+                source_details = {
+                    "database": onboarding_row["bronze_database_{}".format(env)],
```

Comment:
Why not use `f"bronze_database_{env}"` as in `if` above (which IMHO is more pythonic)?